src/Library/demos/Web View/main.js

```diff
@@ -0,0 +1,90 @@
+import WebKit from "gi://WebKit?version=6.0";
+const GObject = imports.gi.GObject;
+const { Uri } = imports.gi.GLib;
```

Comment:
I'll fix it, but please remember to be consistent

`import x from y` should be used